### DISCUSSION TITLE
Shopify Markets

### DISCUSSION THREAD

**Discussion by chrisBradley-dev** [unopim/unopim#70] [Q&A]
Wondering what the best setup would be for shopify markets, where there is one channel but 6-10x fixed prices (for each market)?

I could add some custom price options to a channel (to make it easier to read), but it might be easier to manage on a channel basis

**Comment by devansh-pal-webkul** [comment]
@chris-chameleonbrands,
This requirement appears to be similar to the Shopify Market Prices add-on. We will review it and give it further consideration.

**Comment by itxashancode** [comment]
I'd lean toward separate channels per market - keeps pricing logic isolated, easier to audit, and avoids messy conditional overrides. If you need a single channel view, you could add a `market_prices` JSON field to the product/channel join table and write a small admin UI to manage them, but syncing that to Shopify Markets will still require a custom app. The fixed-price-per-market model fits Shopify's `price_list` API better than variant-level overrides, so you'd get cleaner sync and fewer conflicts.

**Reply by chrisBradley-dev** [reply]
@itxashancode this is an interesting way of looking at it. 

we currently have shopify as a single channel, and then have multiple prices within it, then graphQL updates to shopify from a product level. we are looking to rebuild as there as been some significant structural changes so I will definitely consider this.

**Reply by itxashancode** [reply]
Hope It Helped :3
If the issue is resolved, kindly mark my comment as an answer and close the discussion, and a follow would be appreciated
